### PR TITLE
Add Cartesi Token (CTSI)

### PR DESCRIPTION
**Pull request by colinsteil**

### CODE REVIEW

**Comment by colinsteil** [comment]
No problem, adjusted these. Please let us know if all is good.

File: tokens/eth/0x491604c0FDF08347Dd1fa4Ee062a822A5DD06B5D.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    "social": {
+        "blog": "https://medium.com/cartesi",
+        "facebook": "https://facebook.com/cartesiproject",
+        "github": "https://github.com/cartesi",
+        "discord": "https://discord.gg/Pt2NrnS",
+        "instagram": "https://www.instagram.com/cartesiproject/",
+        "linkedin": "https://www.linkedin.com/company/cartesiproject/",
+        "reddit": "https://www.reddit.com/r/cartesi/",
+        "telegram": "https://t.me/cartesiproject",
+        "telegram announcements": "https://t.me/cartesiannouncements",
```
Comment (ligi) [inline tokens/eth/0x491604c0FDF08347Dd1fa4Ee062a822A5DD06B5D.json:28]: this is not a supported field - please remove

Comment (colinsteil) [reply tokens/eth/0x491604c0FDF08347Dd1fa4Ee062a822A5DD06B5D.json]: Removed telegram announcements. Is Discord ok?

Comment (ligi) [reply tokens/eth/0x491604c0FDF08347Dd1fa4Ee062a822A5DD06B5D.json]: yes

**Comment by colinsteil** [comment]
Cartesi-Official-Icon-LIGHT-Circled (4)

Hey @ligi is it also possible to add our token logo so that it shows up in My Crypto?

Much appreciated. Thanks.

**Pull request merged**